src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx

```diff
@@ -17,42 +17,20 @@ redirects:
 
 We welcome your contributions, whether you are a New Relic employee or a New Relic user! And we don't want you to worry about style. When you edit a file, tech writers on our team review it for style, grammar, and formatting. That said, if you're curious about our style guidelines, you're welcome (but not obligated) [to take a look](/docs/style-guide/get-started/introduction-style-guide/).
 
-<Callout variant="tip">
-  If your draft needs to be released on a specific date or within a specific
-  timeframe (for example, right before a release), contact the Tech Docs `@hero`
-  in the [#documentation Slack
-  channel](https://newrelic.slack.com/messages/documentation/). If you're not a
-  New Relic employee, please [create a GitHub
-  issue](https://github.com/newrelic/docs-website/issues/new/choose).
-</Callout>
-
 ## Edit a doc [#edit-doc]
 
-If you see a minor problem in our documentation that you want to quickly fix, you can use GitHub to edit the file and submit your pull request. A member of the Tech Docs team will review your edit and publish your changes. We'll follow up with you if we have any questions.
-
-<Callout variant="important">
-  You need a GitHub account to edit docs. If you don’t have one, you can [create
-  it here](https://github.com/).
-</Callout>
+If you see a minor problem in our documentation that you want to quickly fix, you can use GitHub to edit the file and submit your pull request. A member of the Docs team will review your edit and publish your changes. We'll follow up with you if we have any questions.
 
 To edit existing content without building the site locally:
 
-1. Find a doc you'd like to edit.
-2. Click **Edit file** on the top corner of the right nav.
-3. In the GitHub file, click on the **pencil** icon within the code block on the top right corner to access the [markdown](https://en.wikipedia.org/wiki/Markdown) editor.
-4. Make your changes.
-5. Write a meaningful commit message and description, then commit your change by creating a new branch.
-6. Follow the prompts to submit your pull request. Please fill in as many details as possible in the provided template.
-7. And you're done!
-
-A member of the Tech Docs team will let you know when we've published your changes.
+1. On the docs site, navigate to the doc you'd like to edit..
```

Comment:
There's two points at the end of the sentence.

Reply:
Yikes, good catch!